app/views/dashboard/index.html.erb

```diff
@@ -16,19 +16,50 @@
         margin-top: 15px;
     }
 
-    .box {
-        width: 350px;
-        border: 1px solid black;
-        padding: 5px;
-        margin: 5px;
-        margin-bottom: 15px;
+    .subtitle {
+        font-family: Roboto, sans-serif;
+        font-weight: bold;
+        font-size: 20px;
+        spacing: auto spacing;
+        color: #b8043c;
+        margin-bottom: 7px;
+    }
+
+    .button {
+        width: 100%;
+        border: none;
+        color: white;
+        padding: 10px 20px;
+        text-align: center;
+        text-decoration: none;
+        display: inline-block;
+        margin: 4px 2px;
+        cursor: pointer;
+        font-family: Roboto, sans-serif;
+        font-weight: bold;
+        font-size: 16px;
+        spacing: auto spacing;
+        margin-bottom: 18px;
+        <% if user_signed_in? && @user.notifications.where(unread: true).size > 0 %>
```

Comment:
Der Check, ob der user signed in ist, sollte überflüssig sein, da nur auf das Dashboard zugegriffen werden kann, wenn man signed in ist.

Reply:
An der Stelle sind ohne zumindest immer Tests fehlgeschlagen, weil es user nicht kannte